plugins/@grouparoo/postgres/__tests__/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "noImplicitAny": false
```

Comment:
such a shame :(

Reply:
I can try a pass into the tests, see if I can get rid of this. This was a stopgap.

Reply:
I think that `noImplicitAny: true` is more valuable than getting linting to work.  Ideally, both can be enabled, but if I had to choose, I would prefer to enable `noImplicitAny` first.

Reply:
oh wait! this is a new `tsconfig.json` just for the tests!

Reply:
I was able to remove this!